docs/services/accountingservice.md

```diff
@@ -0,0 +1,58 @@
+# Accounting Service
+
+This service calculates the total amount of sold products.
```

Comment:
After reading through the code, it seems that there is no logic of "counting the total amount of sold products"

Reply:
Yes, it is currently implemented as mock to keep this PR as small as possible.     
This is documented in the service [READMEs](https://github.com/open-telemetry/opentelemetry-demo/pull/457/files#diff-0c64c2a12cdee702d0bb9380f0301e71908db96de80ecda30fad292155b39b56R4) and the [service table](https://github.com/open-telemetry/opentelemetry-demo/pull/457/files#diff-831e9e9de875f757f893952b18ce744c89d47813416de774ae0c7885dd5407c8R13).